standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -2066,6 +2066,10 @@ struct GetReplicationMetricsRequest {
   3: optional i64 dumpExecutionId
 }
 
+struct GetOpenTxnsRequest {
+  1: required list<TxnType> excludeTxnTypes;
```

Comment:
Could we make this optional? So if we later want to change the GetOpenTxnsRequest object we do not end up sending an empty list all the time? (as a general rule for new methods we create a Request object for them with all optional fields)
Also this might need some checks later in the code, but we might want to merge the codepath for the original get_open_txns method with this new one as soon as possible.

Reply:
Yes makes sense. This pull request is already merged. I will make the param optional in another pull request and will create a ticket to use this new method across and deprecate the original get_open_txns method.

Reply:
https://issues.apache.org/jira/browse/HIVE-23881